docs/Design/Interfaces/Interface Methods.md

```diff
@@ -0,0 +1,116 @@
+# Interface Methods
+
+## Feature Overview
+
+Method declarations within interfaces, as well as variable declarations within interfaces.
+
+
+## Commands
+
+### `interface method : name type [argName1 argType1 argName2 argType2...]`
+
+A command to declare a method within an interface.
+
+
+## Usage
+
+```gls
+interface start : Point
+interface method : method1 int x int

```

Comment:
I like that you're basing this off a real-world example. `method1` doesn't go with that, but something like `getPoint` does (though it looks like you'd want a different one, to take in a parameter).